### DISCUSSION TITLE
Inline Hints Disappear

### DISCUSSION THREAD

**Discussion by danmartinvela** [zed-industries/zed#12845] [Feature Requests]
So I'm working with Rust and then I create another Rust project in the same work directory with cargo new
dir
  proy1
  proy2
  
 after creating proy2 and trying to start coding in proy2 the inlay hints disappear completely... no Rust hints, I need to quit and then reopen the app to get the Rust analyzer back... really weird... been experiencing this for months...
  
 
macOS 14.5
cargo new using the macOS default terminal

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
